subsys/metrics/src/main/conf/conf.d/metrics.conf

```diff
@@ -1,10 +1,35 @@
 [metrics]
 enabled = false
-reporter = graphite.reporter
-reporter.enabled = false
+reporter.enabled = true
+
+######################################
+# List of enabled reporters (comma-separated)
+######################################
+
+reporter = graphite;zabbix
+
+############################################
+# GraphiteDB reporter options
+############################################
+
 graphite.reporter.hostname = localhost
 graphite.reporter.port = 2003
 graphite.reporter.prefix = local.one
 graphite.reporter.simple.period = 30
 graphite.reporter.jvm.period = 300
 graphite.reporter.healthcheck.period = 600
+
+############################################
+# Zabbix reporter options
+############################################
+
+zabbix.reporter.api.host.url = example.com
+zabbix.reporter.host.port = 10051
+zabbix.reporter.host = 192.168.1.1
+zabbix.reporter.user = test
+zabbix.reporter.pwd = test
+zabbix.reporter.local.hostname = example.com
+zabbix.reporter.prefix = local.one
+zabbix.reporter.simple.period = 30
```

Comment:
This is one example of the need for documentation...I don't know what this period controls.